docs/content/developers/index.md

```diff
@@ -0,0 +1,56 @@
+# Technische documentatie voor developers
+
+De standaard wordt opgebouwd op basis van drie componenten, elk met hun eigen
+OAS 3.0 definities. Het gaat dan om de zaakregistratiecomponent (ZRC),
+documentregistratiecomponent (DRC) en de zaaktypecatalogus (ZTC).
+
+De ontwikkeling van iedere component vindt plaats in een aparte git repository:
+
+* [ZRC](https://github.com/vng-Realisatie/gemma-zaakregistratiecomponent)
+* [DRC](https://github.com/VNG-Realisatie/gemma-documentregistratiecomponent)
+* [ZTC](https://github.com/VNG-Realisatie/gemma-zaaktypecatalogus)
+
+## Ontwikkelen van een applicatie die van deze componenten gebruik maakt
+
+Bij het ontwikkelen van vakapplicaties (ook wel afhandelapplicaties genoemd),
+kunnen deze referentie-implementaties ingezet worden als backend services.
+
+### Services starten
+
+Alle componenten zijn als Docker containers beschikbaar.
+
+In `infra` staat een bruikbare `docker-compose.yml` docker-compose file.
+Hiervoor heb je nodig:
+
+* Docker
+* Docker-compose
+
+Voorbeeld gebruik:
+
+```bash
+cd infra
+docker-compose up -d
+```
+
+Je kan nu de componenten bereiken:
+
+* ZRC: http://localhost:8000
+* DRC: http://localhost:8001
+* ZTC: http://localhost:8002
+
+Om de services te stoppen, gebruik:
+
+```bash
+docker-compose down
+```
+
+### Superuser aanmaken
+
+Je kan een superuser aanmaken voor elke service met:
+
+```bash
+docker-compose run (zrc_web|drc_web|ztc_web) python src/manage.py createsuperuser
+```
+
+Vervolgens kan je daarmee inloggen op http://localhost:800x/admin/ om testdata
```

Comment:
Admin werkt nog niet, maar noem vooral ook de API urls even.

Reply:
Wil je hier de URLs noemen, of ze oplijsten in de `demo.html`? Dat laatste lijkt me makkelijker om in sync te houden, dan hoeft een update binnen de component niet een change in deze documentatie te betekenen.